services/core/shared/core/compat/sdk_v5/transactions.js

```diff
@@ -208,7 +208,10 @@ const getTransactions = async params => {
 	return transactions;
 };
 
+const getTransactionsByBlockId = blockId => getTransactions({ blockId });
```

Comment:
`getTransactionsByBlockId` could be bettered by setting the `limit` param in the request as well, for scenarios where the blocks contain over 10 transactions (default limit value). Let us discuss this offline.

Reply:
Refactor `getTransactionsByBlockId` to fetch new block and transactions directly from `lisk-core`.